src/main/kotlin/io/github/nomisrev/routes/articles.kt

```diff
@@ -24,6 +42,18 @@ data class Article(
   val tagList: List<String>
 )
 
+@Serializable
+data class MultipleArticlesResponse(
+  val articles: List<Article>,
+  val articlesCount: Int,
+)
+
+@Serializable
+data class UserFeed(
+  val limit: Long,
+  val offset: Long,
+)
```

Comment:
This is not being used, since we need to pass `limit` and `offset` as query parameters.

 ```suggestion
```